SparkPost/2web2ui
# TR-1282: Lint AMP HTML in prototype fullscreen code editor

This is a subtask, [TR-1282](https://jira.int.messagesystems.com/browse/TR-1282).

### What Changed

- Display AMP validator errors on AMP HTML tab

<img width="1390" alt="Screen Shot 2019-05-29 at 6 32 34 PM" src="https://user-images.githubusercontent.com/1335605/58595900-28784800-8240-11e9-92f7-c839ede3e154.png">


### How To Test
- In "AMP HTML" tab, blank content should render erros.
- In "AMP HTML" tab, copy and paste the following content.  An error should not appear.

```
<!doctype html>
<html ⚡4email>
<head>
  <meta charset="utf-8">
  <script async src="https://cdn.ampproject.org/v0.js"></script>
  <style amp4email-boilerplate>body{visibility:hidden}</style>
</head>
<body>
  <amp-img src="https://example.com/{{company}}.jpg" height="675" width="900"></amp-img>
</body>
</html>
```
